src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid $main-border-color;
+  border-radius: 5px;
+  transition: transform 0.3s;
+
+  &__product-image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
+    transition: color 0.3s;
```

Comment:
The same separation rule applies here between `&__product-title` and `&__product-code`. An empty line should be added to separate these elements.